nilearn/plotting/displays.py

```diff
@@ -946,6 +1024,134 @@ def draw_cross(self, cut_coords=None, **kwargs):
+
+        # safety checks
+        if 's' in node_kwargs:
+            raise ValueError("Please use 'node_size' and not 'node_kwargs' "
+                             "to specify node sizes")
+        if 'c' in node_kwargs:
+            raise ValueError("Please use 'node_color' and not 'node_kwargs' "
+                             "to specify node colors")
+
+        adjacency_matrix_shape = adjacency_matrix.shape
+        if (len(adjacency_matrix_shape) != 2 or
+                adjacency_matrix_shape[0] != adjacency_matrix_shape[1]):
+            raise ValueError(
+                "'adjacency_matrix' is supposed to have shape (n, n)."
+                ' Its shape was {0}'.format(adjacency_matrix_shape))
+
+        node_coords_shape = node_coords.shape
+        if len(node_coords_shape) != 2 or node_coords_shape[1] != 3:
+            raise ValueError(
+                "'node_coords' should be a array with shape (n, 3). "
+                'Its shape was {0}'.format(node_coords_shape))
+
+        if node_coords_shape[0] != adjacency_matrix_shape[0]:
+            raise ValueError(
+                "Shape mismatch between 'adjacency_matrix' "
+                "and 'node_coords'"
+                "'adjacency_matrix' shape is {0}, 'node_coords' shape is {1}"
+                .format(adjacency_matrix_shape, node_coords_shape))
+
+        if sparse.issparse(adjacency_matrix):
+            adjacency_matrix = adjacency_matrix.toarray()
+
+        if not np.allclose(adjacency_matrix, adjacency_matrix.T):
+            raise ValueError("'adjacency_matrix' should be symmetric")
+
+        # For a masked array, masked values are replaced with zeros
+        if hasattr(adjacency_matrix, 'mask'):
+            adjacency_matrix = adjacency_matrix.filled(0)
+
+        if edge_threshold is not None:
+            if isinstance(edge_threshold, basestring):
+                message = ("If 'edge_threshold' is given as a string it "
+                           'should be a number followed by the percent sign, '
+                           'e.g. "25.3%"')
+                if not edge_threshold.endswith('%'):
+                    raise ValueError(message)
+
+                try:
+                    percentile = float(edge_threshold[:-1])
+                except ValueError as exc:
+                    exc.args += (message, )
+                    raise
+
+                # Keep a percentile of edges with the highest absolute
+                # values, so only need to look at the covariance
+                # coefficients below the diagonal
+                lower_diagonal_indices = np.tril_indices_from(adjacency_matrix,
+                                                              k=-1)
+                lower_diagonal_values = adjacency_matrix[
+                    lower_diagonal_indices]
+                lower_diagonal_abs_values = np.abs(lower_diagonal_values)
+                edge_threshold = fast_abs_percentile(
+                    lower_diagonal_abs_values, percentile)
```

Comment:
Doesn't the fast_abs_percentile already do the abs? In other words, can't you just pass it simply lower_diagonal_values?